### PR TITLE
Geneva Exporter Release v1.2.4

### PR DESCRIPTION
**Pull request by mic-max**
Please review @open-telemetry/dotnet-maintainers :)

### CODE REVIEW

File: src/OpenTelemetry.Exporter.Geneva/CHANGELOG.md [hunk 1]
```diff
@@ -1,6 +1,9 @@
-# Changelog
+# Changelog - OpenTelemetry.Exporter.Geneva
 
-## Unreleased
+## 0.0.1 [2022-Apr-19]
+
+This is teh first release for the `OpenTelemetry.Exporter.Geneva`
```
Comment (reyang) [inline src/OpenTelemetry.Exporter.Geneva/CHANGELOG.md:5]:
```suggestion
This is the first release for the `OpenTelemetry.Exporter.Geneva`
```

Comment (reyang) [reply src/OpenTelemetry.Exporter.Geneva/CHANGELOG.md:5]: Or maybe 

```suggestion
This is a very first test release for the `OpenTelemetry.Exporter.Geneva`
```

**Review by reyang — APPROVED** [review]
LGTM assuming the nits get fixed.

File: src/OpenTelemetry.Exporter.Geneva/CHANGELOG.md (hunk 1 above)
Comment (utpilla) [reply src/OpenTelemetry.Exporter.Geneva/CHANGELOG.md:5]: I think we could use "This is the first test release for ..."

File: src/OpenTelemetry.Exporter.Geneva/CHANGELOG.md [hunk 2]
```diff
@@ -1,6 +1,9 @@
-# Changelog
+# Changelog - OpenTelemetry.Exporter.Geneva
```
Comment (utpilla) [inline src/OpenTelemetry.Exporter.Geneva/CHANGELOG.md:1]: You don't need to add the project name here as this CHANGELOG file already exists under the directory `OpenTelemetry.Exporter.Geneva`

Look at other projects in the solution for reference to stay consistent.

File: src/OpenTelemetry.Exporter.Geneva/CHANGELOG.md [hunk 3]
```diff
@@ -1,6 +1,9 @@
...
 
-## Unreleased
```
Comment (utpilla) [inline src/OpenTelemetry.Exporter.Geneva/CHANGELOG.md:3]: Don't remove the `## Unreleased` section. We always have it at the top of the CHANGELOG

File: src/OpenTelemetry.Exporter.Geneva/CHANGELOG.md (hunk 2 above)
Comment (reyang) [reply src/OpenTelemetry.Exporter.Geneva/CHANGELOG.md:1]: Looks like others are doing this https://github.com/open-telemetry/opentelemetry-dotnet-contrib/blob/main/src/OpenTelemetry.Contrib.Extensions.AWSXRay/CHANGELOG.md#changelog---opentelemetrycontribextensionsawsxray

Comment (reyang) [reply src/OpenTelemetry.Exporter.Geneva/CHANGELOG.md:1]: And it is very inconsistent just by quickly looking at 3 projects.
I do agree that it's better to just keep it as "Changelog".

Comment (utpilla) [reply src/OpenTelemetry.Exporter.Geneva/CHANGELOG.md:1]: Ahh I wasn't aware of this earlier.

Comment (mic-max) [reply src/OpenTelemetry.Exporter.Geneva/CHANGELOG.md:1]: Ya was just going off another project but if that's not something followed everywhere I agree to omit the project name :)

**Review by cijothomas — APPROVED** [review]

**Pull request merged**